introducing-java/src/main/java/com/onto/javacourse/intro/arrayReverse/ArrayReverse.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * A utility class for reversing array
+ */
+public class ArrayReverse {
+
+    /**
+     * Reverses the elements of the given array.
```

Comment:
This is misleading. Array is not reverse, instead you do some thing else. Can you try to find out what is that?